src/index.ts

```diff
@@ -71,17 +71,37 @@ export function resolveBundle(bundle: Bundle, resolvers: Resolver[]): Promise<Re
 			// Now that we have a resolver, add the new files needed to the stream
 			resolver.resolve(bundle)
 			.then((additionalItems) => {
-				additionalItems.map((file: FileInfo) => {
+
+				return Promise.map(additionalItems, (file: FileInfo) => {
 					pack.entry({ name: file.name, size: file.size }, file.contents)
+					if (file.name === 'Dockerfile') {
+						return bundle.dockerfileHook(file.contents.toString())
```

Comment:
Yeah good point, especially as I hide the `dockerfileContents` behind a getter in the same PR...